src/components/Global/Navbar.js

```diff
@@ -24,14 +24,14 @@ const Navbar = () => {
           <div className="flex flex-1 items-center">
             <div className="flex w-full items-center justify-between md:w-auto">
               <Link href={'/'}>
-                <>
-                  <span className="sr-only">WebX DAO</span>
+                <a href="/">
+                  <span className="sr-only ">WebX DAO</span>
```

Comment:
This might be causing the issue.
```suggestion
                  <span className="sr-only">WebX DAO</span>
```